### PR TITLE
[Enhancement] Add default avatar when profile image is not available

### LINKED ISSUE #164: Set default avatar image if profile image is not available
- [ ] Create a component to display the first two letters of the name of the agent

### PR DESCRIPTION
**Pull request by josemiguelo**
image

New component created for default Avatar when Agent thumbnail is not available
Fixes #164

### CODE REVIEW

**Comment by josemiguelo** [comment]
@pranavrajs please take a look. Thanks!

**Comment by pranavrajs** [comment]
@josemiguelo Nice work! Thanks for the PR.

File: app/javascript/dashboard/components/widgets/Avatar.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        borderRadius: this.rounded ? '50%' : 0,
+        lineHeight: `${this.size + Math.floor(this.size / 20)}px`,
+        fontWeight: 'bold',
+        alignItems: 'center',
+        justifyContent: 'center',
+        textAlign: 'center',
+        backgroundColor: this.backgroundColor,
+        font: `${Math.floor(this.size / 2.5)}px/${
+          this.size
+        }px Helvetica, Arial, sans-serif`,
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/Avatar.vue]: Could you please remove font family from here? I would like to have font-family inherited from body.

File: app/javascript/dashboard/components/widgets/Avatar.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  },
+  computed: {
+    style() {
+      return {
+        display: 'flex',
+        width: `${this.size}px`,
+        height: `${this.size}px`,
+        borderRadius: this.rounded ? '50%' : 0,
+        lineHeight: `${this.size + Math.floor(this.size / 20)}px`,
+        fontWeight: 'bold',
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/Avatar.vue]: Can we move the constant attributes to class
```
<style lang="scss">
.avatar--container {
  display:flex // etc
}
</style>
```

Comment (josemiguelo) [reply app/javascript/dashboard/components/widgets/Avatar.vue]: Done!

File: app/javascript/dashboard/components/widgets/Avatar.vue (hunk 1 above)
Comment (josemiguelo) [reply app/javascript/dashboard/components/widgets/Avatar.vue]: Of course. Now the look of the avatar is coherent with the rest of the screen.
Done!

**Comment by sojan-official** [comment]
Thanks a lot for PR.

Ps: We're sending stickers to our contributors ❤️ Let us know your details https://chatwoot.typeform.com/to/ihG9Gz

**Comment by pranavrajs** [comment]
@josemiguelo This is how it looks in production. Nice work! Thanks.

Screenshot 2019-10-29 23 48 57

**Pull request merged**